### PR TITLE
Added overwrite warnings and improved user interface

### LINKED ISSUE #14: Add overwrite warnings to annotate_dir
If using annotate_dir, the program will end if any of the texts or masks already exist, I need to implement a system for warning the user and overwriting them.

### PR DESCRIPTION
**Pull request by msekatchev**
Added overwrite warnings for saving text and mask files with annotate_dir
-	If the file already exists, user is prompted to overwrite it or delete the new file.
-	Closes #14 (Add overwrite warnings to annotate_dir).
Improved user interface:
-	Improved print() information messages, removing separation when outputting strings directly followed by punctuation.
-	Added an arrow at the start of a line requiring user input.

### CODE REVIEW

**Review by pdeperio — CHANGES_REQUESTED** [review]
Is the delete option necessary if you're going to overwrite anyways?

**Comment by msekatchev** [comment]
> Is the delete option necessary if you're going to overwrite anyways?

Yeah `shutil.move(textName,text_save_path)` will throw an error if the file already exists, unless the full directory name is specified (see [here](https://stackoverflow.com/questions/31813504/move-and-replace-if-same-file-name-already-exists)).

**Comment by pdeperio** [comment]
ah sorry, i didn't look at the code yet. you don't explicitly delete it yourself. what did you mean by "or delete the new file" then?

File: annotate.py [hunk 1]
```diff
@@ -225,10 +226,45 @@ def annotate_dir(img_dir, initials, size1=1, size2=1) :
...
+
+
+        print("\n")
+
+
+        saveMask=True
+        if(os.path.exists(os.path.join(mask_save_path,maskName))):
+            print("File", maskName, "already exists in", mask_save_path+". Would you like to overwrite it?")
+            overwriteMask = input("(y/n)\n-->")
+            if(overwriteMask == "y" or overwriteMask == "Y"):
```
Comment (pdeperio) [inline annotate.py]: for case-insensitivity, use the `.lower()` function ([ref](https://stackoverflow.com/questions/319426/how-do-i-do-a-case-insensitive-string-comparison)).

line 234 above too

**Review by pdeperio — CHANGES_REQUESTED** [review]

**Comment by msekatchev** [comment]
> ah sorry, i didn't look at the code yet. you don't explicitly delete it yourself. what did you mean by "or delete the new file" then?

If the file already exists in that directory, and we don't want to overwrite it, then the new text or mask that we created will just get deleted.

File: annotate.py (hunk 1 above)
Comment (msekatchev) [reply annotate.py]: Change added.

**Review by pdeperio — APPROVED** [review]

**Pull request merged**